adhocracy4/contrib/views.py

```diff
@@ -18,3 +19,20 @@ def raise_exception(self):
         limited by the current phase.
         """
         return self.request.user.is_authenticated()
+
+
+class SortableListView(generic.ListView):
+    """
+    ordering_current: List of possible ordering tuples (ordering,
+        ordering name).
+    """
```

Comment:
I don't understand how this docstring is related to the code. `ordering_current` does not exist, or does it?